devel/git-extras/files/macports-release.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/bin/git-extras b/bin/git-extras
+index 33098fcfaabd..28fc2056718f 100755
+--- a/bin/git-extras
++++ b/bin/git-extras
+@@ -4,13 +4,12 @@ VERSION="7.1.0-dev"
+ INSTALL_SCRIPT="https://raw.githubusercontent.com/tj/git-extras/master/install.sh"
+ 
+ update() {
+-  local bin="$(command -v git-extras)"
+-  local prefix=${bin%/*/*}
+-  local orig=$PWD
+-
+-  curl -s $INSTALL_SCRIPT | PREFIX="$prefix" bash /dev/stdin \
+-    && cd "$orig" \
+-    && echo "... updated git-extras $VERSION -> $(git extras --version)"
++  echo "This git-extras installation is managed by MacPorts."
++  echo "If you'd like to update git-extras, run the following:"
++  echo
++  echo "  port upgrade git-extras"
```

Comment:
The upgrade command might be better written if it included `sudo`?

```suggestion
+  echo "  sudo port upgrade git-extras"
```

Reply:
I wasn't sure, so I had left it off. Easy to add it and done.